content/accessibility/tools.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Accessibility tools
+---
+
+A selection of tools to help product designers design accessibly. 
+
+## Figma 
+
+### [Accessibility sticker sheet](https://www.figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1)
+This Figma file includes annotations to use on your designs prior to developer hand off. These annotations let you note accessibility information like desired tab order, screen reader labels, headings, and landmarks. Pull directly from the file or turn on the accessibility annotations in your Figma assets list.
+
+![""](https://user-images.githubusercontent.com/40274682/102533171-e6993780-4059-11eb-8e7c-5452070e2de0.png)
+
+### [A11y - Focus Orderer plugin](https://www.figma.com/community/plugin/731310036968334777/A11y---Focus-Orderer)
+
+Select your elements in the desired tabbing order, add them to the plugin to get a visible layer label, and use the tab key to test the tab order from inside of Figma. 
+**How does this help?** Instead of manually placing tab order stickers on your design, let the plugin do it for you, and you can tab through your design to see if the flow makes sense.
+
+![""](https://user-images.githubusercontent.com/40274682/102416403-9dd87480-3fae-11eb-98c3-1b85046d98b7.png)
+
+### [Contrast plugin](https://www.figma.com/community/plugin/748533339900865323/Contrast)
+
+This plugin will check the contrast of text, graphical objects, and even gradients. 
+**How does this help?** Checking the color contrast is one of the most important steps to take as a designer. Now you can test contrast without leaving the design tool. 
+
+![""](https://user-images.githubusercontent.com/40274682/88444681-1c036700-cdd3-11ea-9be2-92eb651419d3.png)
+
+## System tools
+
+[Sim Daltonism](https://apps.apple.com/us/app/sim-daltonism/id693112260?mt=12): Free application in the Mac App Store that can simulate various color blindness disorders.
+**How does this help?** When you're designing something that uses color (yes, that's everything), you can use this app to see the design as someone who has color blindness would see it, and check if it still makes sense.
+
+![""](https://user-images.githubusercontent.com/40274682/88444732-5ff66c00-cdd3-11ea-97f6-350c2f5a6f32.png)
+
+
+## Web tools
+### Color contrast checker
+
+[Color Review](https://color.review/): Color review is fantastic if you're looking for help finding an accessible color in an existing palette. 
+
+![""](https://user-images.githubusercontent.com/40274682/88444756-90d6a100-cdd3-11ea-876c-1c850ae19223.png)
```

Comment:
### Photoshop and color LUTs (Look up tables) for color vision deficiency

While Photoshop has several color preview modes for color vision deficiency it's not possible to copy the processed image this way, as the color diff is processed internally in the graphics card. Permanent color conversions to a file should be applied by using color adjustment layers and different color LUTs.

[Applying LUTs in Photoshop](http://blog.eppz.eu/design-color-blind-users/)